competition-simulator.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is not the same as the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulator is based off [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.4.
+
+### Installing the simulator
```

Comment:
Should we use a different term than "simulator" here? As I understand it, Webots is doing the actual simulation -- the extra files we're providing are more a description of the world.

In particular it reads oddly to me that below we specify "Download the simulator" but then instruct to use the Webots IDE to open that simulator.

I think it's because to me a "simulator" is something which you'd expect to run, rather than work with.

Even just "simulation" might be a clearer framing as that's perhaps clearer that it's a thing you're going to work with.

Reply:
I like the idea of using the word "simulation" instead of "simulator" to refer to our project in WeBots

Reply:
Have addressed in the places it makes sense